src/index.js

```diff
@@ -1,26 +1,36 @@
-import grapesjs from 'grapesjs';
+import grapesjs from "grapesjs";
 
-export default grapesjs.plugins.add('gjs-blocks-basic', (editor, opts = {}) => {
+export default grapesjs.plugins.add("gjs-blocks-basic", (editor, opts = {}) => {
   const config = {
-    blocks: ['column1', 'column2', 'column3', 'column3-7', 'text', 'link', 'image', 'video', 'map'],
+    blocks: [
+      "column1",
+      "column2",
+      "column3",
+      "column3-7",
+      "text",
+      "link",
+      "image",
+      "video",
+      "map"
+    ],
     flexGrid: 0,
-    stylePrefix: 'gjs-',
+    stylePrefix: "gjs-",
     addBasicStyle: true,
-    category: 'Basic',
-    labelColumn1: '1 Column',
-    labelColumn2: '2 Columns',
-    labelColumn3: '3 Columns',
-    labelColumn37: '2 Columns 3/7',
-    labelText: 'Text',
-    labelLink: 'Link',
-    labelImage: 'Image',
-    labelVideo: 'Video',
-    labelMap: 'Map',
+    category: "Basic",
+    labelColumn1: "1 Column",
+    labelColumn2: "2 Columns",
+    labelColumn3: "3 Columns",
+    labelColumn37: "2 Columns 3/7",
+    labelText: "Text",
+    labelLink: "Link",
+    labelImage: "Image",
+    labelVideo: "Video",
+    labelMap: "Map",
+    rowHeight: 75,
     ...opts
   };
 
   // Add blocks
-  const loadBlocks = require('./blocks');
+  const loadBlocks = require("./blocks");
```

Comment:
All single quotes were changed in double-quotes, is it intended o something related to your editor? In any case, I'd like to keep single quotes

Reply:
I'm sorry, I don't realize that my editor was changed for double-quotes.... I just put the single-quotes back